### ISSUE TITLE
Start page example in GOV.UK Frontend review application has illogical layout

### ISSUE THREAD

**Issue by nickcolley** [alphagov/govuk-frontend#1415]
This issue is from a May 2019 external accessibility audit report.

**WCAG Reference**: Usability feedback only, there is no WCAG related guidelines.
**Issue ID**: DAC_Issue31
**URL**: http://govuk-frontend-v3.herokuapp.com/full-page-examples/start-page

## Screen Shot
![](https://user-images.githubusercontent.com/2445413/58716574-402b0a00-83c1-11e9-9557-b6c0aeb5464d.png)

The start pages contain important information that appears below the ‘Start now’ button.
This could result in users missing important content.

## Current Code Ref(s)
```html
<a href="#" role="button" draggable="false" class="govuk-button govuk-button--start govuk-!-margin-top-2 govuk-!-margin-bottom-8"> Start now </a> <div class="govuk-tabs" data-module="tabs"> <h2 class="govuk-tabs__title">Contents </h2> <ul class="govuk-tabs__list" role="tablist"> <li class="govuk-tabs__list-item" role="presentation"> <a class="govuk-tabs__tab govuk-tabs__tab--selected" href="#more-information" id="tab_more-information" role="tab" aria-controls="more-information" tabindex="0" aria-selected="true"> More information </a> </li>
```

## Screen reader comments
> “The additional information to provide me with more information was located below the ‘Start now’ button. This could be problematic for some users as many users might not navigate past the ‘Start now’ button to view more content and may need to return to the start page to locate the further information. Ensuring that all necessary information to complete the process can be located above the ‘Start now’ button will enable users to complete the service with all necessary information available prior to beginning the service. It may also assist users if headings could be used for the additional information rather than tabs to ensure that the user is able to locate the information more efficiently. This occurred using JAWS, NVDA and VoiceOver for iPhone and Mac.”

## Solution
Consider placing the information prior to the ‘Start now’ button, or on a page prior to the ‘Start now’ button. It would also benefit users if a heading could be provided above the ‘More information’ text, stating words to the effect of ‘Before you start’.

**Comment by nickcolley** [comment]
I think given this is usability feedback on the [start pages pattern](https://design-system.service.gov.uk/patterns/start-pages/) owned by GOV.UK Publishing we should record this in the [start pages backlog entry](https://github.com/alphagov/govuk-design-system-backlog/issues/111) and let them know.

I've added awaiting-triage for us to discuss this as a team
